analytics/terraform/emr-eks-ack-crossplane/addons.tf

```diff
@@ -19,19 +19,7 @@ module "eks_blueprints_kubernetes_addons" {
   #   Further tuning for CoreDNS is to leverage NodeLocal DNSCache -> https://kubernetes.io/docs/tasks/administer-cluster/nodelocaldns/
   #---------------------------------------------------------
   enable_coredns_autoscaler = true
```

Comment:
You can remove this as well. `coredns_autosclaer` is enabled by default in EKS Blueprints

Reply:
removed